packages/vscode-extension/package.nls.json

```diff
@@ -258,18 +258,32 @@
     "teamstoolkit.localDebug.switchM365AccountWarning": "You are now using a different Microsoft 365 tenant from what you previously used.",
     "teamstoolkit.localDebug.taskDefinitionError": "The value of '%s' is invalid for the task of type 'teamsfx'",
     "teamstoolkit.localDebug.taskCancelError": "The task is canceled.",
-    "teamstoolkit.localDebug.multipleTunnelServiceError": "There are multiple running local tunnel tasks. Please close the redundant tasks.",
-    "teamstoolkit.localDebug.noTunnelServiceError": "There is no running local tunnel tasks. Please start the local tunnel task first.",
+    "teamstoolkit.localDebug.multipleTunnelServiceError": "There are multiple running local tunneling services. Close any redundant tasks to prevent conflicts.",
+    "teamstoolkit.localDebug.noTunnelServiceError": "There is no running local tunneling service. Ensure that the local tunneling service has been started.",
     "teamstoolkit.localDebug.ngrokStoppedError": "Ngrok has been stopped with exit code '%s'.",
     "teamstoolkit.localDebug.ngrokProcessError": "Failed to start ngrok. %s",
     "teamstoolkit.localDebug.ngrokNotFoundError": "Ngrok is not installed by TeamsFx. See teamsfx-debug-tasks#debug-check-prerequisites for how to install the ngrok.",
     "teamstoolkit.localDebug.ngrokInstallationError": "Failed to install Ngrok. %s",
-    "teamstoolkit.localDebug.tunnelServiceNotStartedError": "The tunnel service hasn't been started. Please wait for a moment or restart the local tunnel task.",
-    "teamstoolkit.localDebug.tunnelEndpointNotFoundError": "Failed to find the tunnel endpoint. Teams toolkit try to get the first HTTPS URL from %s and failed.",
+    "teamstoolkit.localDebug.tunnelServiceNotStartedError": "The tunneling service hasn't been started. Wait for a moment or restart the local tunneling task.",
+    "teamstoolkit.localDebug.tunnelEndpointNotFoundError": "Failed to find the tunnel endpoint. Teams toolkit tried to get the first HTTPS URL from %s and failed.",
     "teamstoolkit.localDebug.tunnelEnvError": "Failed to save environment variables. %s",
+    "teamstoolkit.localDebug.startTunnelError": "Task to start local tunneling service failed. %s",
```

Comment:
Separate the two sentences if '%s' has no relationship with this string. Or it'll be confusing to localization team.

Reply:
Thx, I will create another PR to change all the `%s` in localdebug